x-pack/plugins/infra/public/components/asset_details/tabs/overview/metrics/metrics_grid.tsx

```diff
@@ -4,127 +4,79 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import React, { useCallback } from 'react';
+import React from 'react';
 
-import { EuiFlexGrid, EuiFlexItem, EuiTitle, EuiSpacer, EuiFlexGroup } from '@elastic/eui';
+import { EuiSpacer } from '@elastic/eui';
 import type { DataView } from '@kbn/data-views-plugin/public';
 import type { TimeRange } from '@kbn/es-query';
-import { FormattedMessage } from '@kbn/i18n-react';
-import { LensEmbeddableInput } from '@kbn/lens-plugin/public';
 import {
-  assetDetailsDashboards,
-  XY_MISSING_VALUE_DOTTED_LINE_CONFIG,
-} from '../../../../../common/visualizations';
-import { buildCombinedHostsFilter } from '../../../../../utils/filters/build';
-import { LensChart, HostMetricsExplanationContent } from '../../../../lens';
-import { METRIC_CHART_HEIGHT } from '../../../constants';
-import { Popover } from '../../common/popover';
-import type { DataViewOrigin } from '../../../types';
-import { useDateRangeProviderContext } from '../../../hooks/use_date_range';
-
-type BrushEndArgs = Parameters<NonNullable<LensEmbeddableInput['onBrushEnd']>>[0];
+  nginxStubstatusMetrics,
+  nginxAccessMetrics,
+} from '../../../../../common/visualizations/lens/dashboards/asset_details/host/nginx_charts';
+import { assetDetailsDashboards } from '../../../../../common/visualizations';
```

Comment:
nit: In `common/visualizations`, we could create `nginxDashboard` object that would contain both  `nginxStubstatusMetrics`, `nginxAccessMetrics` charts. wdyt?

I just noticed that we're naming these chart objects with `Charts` suffix. eg: `hostKPICharts`, `hostMetricCharts`. Could we rename the nginx ones as `nginxStubstatusCharts` and `nginxAccessChart`?

Reply:
Thanks for the suggestion, it makes sense to put them there and rename them to charts, fixed ✅